app/furniture/marketplace/order.rb

```diff
@@ -20,15 +20,17 @@ class Order < Record
     }
 
     def product_total
-      ordered_products.sum(0) do |ordered_product|
-        ordered_product.product.price * ordered_product.quantity
-      end
+      ordered_products.sum(0, &:price_total)
+    end
+
+    def tax_total
+      ordered_products.sum(0, &:tax_amount)
```

Comment:
_:: gestures vaguely in the direction of the 0s ::_

Reply:
<img width="1227" alt="Screenshot 2023-03-10 at 7 21 54 PM" src="https://user-images.githubusercontent.com/50284/224462350-04776b42-d320-431a-9991-a49ca21b4014.png">

Reply:
fine, FINE